prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -470,15 +471,16 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
             .register(self.mir.span, ErrorCtxt::Unexpected);
         let method_with_fold_unfold = foldunfold::add_fold_unfold(
             self.encoder,
-            self.cfg_method,
+            self.cfg_method.clone(),
             &loan_locations,
             &self.cfg_blocks_map,
             method_pos,
         )
         .map_err(|foldunfold_error| {
             SpannedEncodingError::internal(
                 format!(
-                    "generating fold-unfold Viper statements failed ({:?})",
+                    "generating fold-unfold Viper statements for method {:?} failed ({:?})",
+                    self.cfg_method.name(),
```

Comment:
`cfg_method.name()` is the Viper method name, so it's probably unreadable for the user. What about displaying the Rust's method name instead?